README.md

```diff
@@ -2,10 +2,10 @@
 
 [![gitleaks badge](https://img.shields.io/badge/protected%20by-gitleaks-blue)](https://github.com/zricethezav/gitleaks#pre-commit) [![gitsecrets](https://img.shields.io/badge/protected%20by-gitsecrets-blue)](https://github.com/awslabs/git-secrets)
 
-Repository for [Configuration as Code (CasC) for Controllers](https://docs.cloudbees.com/docs/cloudbees-ci/latest/casc-controller/).
+This repository is for [Configuration as Code (CasC) for controllers](https://docs.cloudbees.com/docs/cloudbees-ci/latest/casc-controller/).
 
-Plugins and plugin catalogs curated with [casc-plugin-dependency-calculation](https://github.com/kyounger/casc-plugin-dependency-calculation).
+Plugins and plugin catalogs are curated with the [CloudBees CasC Plugin Catalog and Transitive Dependencies Calculator](https://github.com/kyounger/casc-plugin-dependency-calculation).
```

Comment:
@carlosrodlop - I removed the reference to plugin catalogs here, since I don't think they're supported for OC CasC bundles. If this is incorrect, please revert this change.